paths/workflow_runs.yaml

```diff
@@ -0,0 +1,102 @@
+  workflow_runs:
+    get:
+      summary: A single workflow run can be retrieved by calling this endpoint with the unique identifier of the workflow run.
+      operationId: retrieve_workflow_run
+      parameters:
+        - name: workflow_run_id
+          in: path
+          required: true
+          description: The unique identifier of the workflow run.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A workflow run object.
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/workflow_runs/workflow_run.yaml
+        default:
+          $ref: ../responses/default_error.yaml
+
+    post:
+      summary: Create a workflow run.
+      operationId: create_workflow_run
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                workflow_id:
+                  type: string
+                  required: true
+                  description: The unique identifier for the workflow.
+                applicant_id:
+                  type: string
+                  required: true
+                  description: The unique identifier for the applicant.
+                custom_data:
+                  type: object
+                  description: Optional object with custom input data to be used in the workflow run.
+                link:
+                  type: object
+                  description: Optional object for the configuration of the workflow run link.
+      responses:
+        "201":
+          description: A workflow run object.
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/workflow_runs/workflow_run.yaml
+        default:
+          $ref: ../responses/default_error.yaml
+
+  task:
+    get:
+      summary: A single task can be retrieved by calling this endpoint with the unique identifier of the task and workflow run.
+      operationId: retrieve_task
+      parameters:
+        - name: workflow_run_id
+          in: path
+          required: true
+          description: The unique identifier of the workflow run to which the task belongs.
+          schema:
+            type: string
+        - name: task_id
+          in: path
+          required: true
+          description: The identifier of the task you want to retrieve.
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A task object.
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/tasks/task.yaml
+        default:
+          $ref: ../responses/default_error.yaml
+
+  tasks:
```

Comment:
super small detail, but I think from a hierarchy perspective the "list" resource endpoint should come before the "retrieve"